src/ape/api/accounts.py

```diff
@@ -215,29 +223,31 @@ def deploy(
     def check_signature(
         self,
         data: Union[SignableMessage, TransactionAPI],
-        signature: Optional[_Signature] = None,  # TransactionAPI doesn't need it
+        signature: Optional[MessageSignature] = None,  # TransactionAPI doesn't need it
     ) -> bool:
         """
         Verify a message or transaction was signed by this account.
 
         Args:
             data (Union[:class:`~ape.types.signatures.SignableMessage`, :class:`~ape.api.transactions.TransactionAPI`]):  # noqa: E501
               The message or transaction to verify.
-            signature (Optional[``_Signature``]): The :class:`~ape.types.signatures.MessageSignature` or the
-              :class:`~ape.types.signatures.TransactionSignature`.
+            signature (Optional[``_Signature``]): The :class:`~ape.types.signatures.MessageSignature`.
```

Comment:
Mostly fixing the type here but also making the link happen at in the type-doc instead of the description.

```suggestion
            signature (Optional[:class:`~ape.types.signatures.MessageSignature`]): The signature.
```